### ISSUE TITLE
cudaSuccess (4 vs. 0)  unspecified launch failure

### ISSUE THREAD

**Issue by pauldelmusica** [msracver/Deep-Image-Analogy#10]
I am having problems running deep_image_analogy.exe in the distribution and my own compiled exe without CUDNN. I am running on Windows 10 with CUDA 8, VS 2013, and GTX 1070.

For the included exe, I got error message:
.........
Finding nearest neighbor field using PatchMatch Algorithm at layer:conv2_1.
F0609 20:49:02.585299 11040 cudnn_conv_layer.cu:28] Check failed: status == CUDNN_STATUS_SUCCESS (8 vs. 0)  CUDNN_STATUS_EXECUTION_FAILED
*** Check failure stack trace: ***

Suspecting CUDNN, I disabled and rebuilt the exe. The new error I got occurs in different *.cu files depending on if I run the program in the /Release folder or if I copy the exe to the /exe folder. Both complain about "cudaSuccess (4 vs. 0)  unspecified launch failure". The messages are attached below:

Finding nearest neighbor field using PatchMatch Algorithm at layer:conv2_1.
F0609 21:23:53.790262  5948 im2col.cu:61] Check failed: error == cudaSuccess (4 vs. 0)  unspecified launch failure
*** Check failure stack trace: ***

Finding nearest neighbor field using PatchMatch Algorithm at layer:conv1_1.
Saving flow result.
Finished finding ann. Time : 27.89
WARNING: Logging before InitGoogleLogging() is written to STDERR
F0609 21:15:16.394630  5436 syncedmem.hpp:31] Check failed: error == cudaSuccess (4 vs. 0)  unspecified launch failure
*** Check failure stack trace: ***

**Comment by pauldelmusica** [comment]
deep_image_analogy.exe ../models/ ../demo/content.png ../demo/style.png ../demo/output/ 0 0.2 2 0

changing ratio from 0.5 to 0.2 gets the program to run.

**Comment by dany-on-demand** [comment]
Can you fire up some gpu analysis tool and check memory usage? Mine never goes above 1.8Gb and I still get this error

Did you compile with or without cuDNN? Did you use CUDA 7.5 or 8.0?

**Comment by Callidior** [comment]
I also can't run the application due to memory problems (presumably) on my GeForce GTX 1050, which has 4 GB of GPU memory. Lowering the ratio works, since it will scale down to images before passing them through the network, but the results are pretty bad.

On a Tesla K40c with 12 GB, everything works fine. However, the memory consumption there seems to never exceed 1 GB, so I wonder why I can't run it on my GPU.

**Comment by dany-on-demand** [comment]
@Callidior Yep I "solved" my problem the same way, a Tesla K80 it can even handle maximum resolution at 1.0 ratio

I've heard about memory spikes, maybe it does run out of memory but it does so quickly the sensor app doesn't pick it up?

Well I'm glad actually, at least this confirms that this is not a temp throttling issue

**Comment by rozentill** [comment]
See if disable the time limit (TDR) would help https://github.com/msracver/Deep-Image-Analogy/issues/25
